src/index.tsx

```diff
@@ -30,6 +30,7 @@ export type {
 } from 'state/swap'
 export { Field } from 'state/swap'
 export type { Slippage } from 'state/swap/settings'
+export type { Transaction, TransactionInfo, TransactionType } from 'state/transactions'
```

Comment:
Should this include Swap, ExactInput, ExactOutput, Wrap, and Unwrap transactions?

Reply:
I didn't need them, but I can't think of a reason not to.